app/test/unit/git/reflog-test.ts

```diff
@@ -67,4 +69,33 @@ describe('git/reflog', () => {
       expect(branches).toContain('branch-3')
     })
   })
+
+  describe('getCheckoutsAfterDate', () => {
+    it('returns does not return the branches that were checked out before a specific date', async () => {
+      await createAndCheckout(repository!, 'branch-1')
```

Comment:
:art: if you change line 25 from

```ts
let repository: Repository | null = null
```

to 

```ts
let repository: Repository
```

You should be able to cleanup all the `!` usages in this test suite